.github/workflows/test-and-publish-nightly.yml

```diff
@@ -51,21 +59,14 @@ jobs:
     container:
       image: ${{ matrix.container }}
       options: --privileged
-    name: "test (${{ matrix.runson.display }}, ${{ matrix.java-distribution }} ${{ matrix.java-version }})"
+    name: "test (${{ matrix.runson.display }}, ${{ matrix.java-distribution }} ${{ matrix.java-version }}${{ matrix.architecture && ' ' || '' }}${{ matrix.architecture }}"
```

Comment:
I guess this change won't be needed if `matrix.runson.display` includes architecture.

Reply:
See comment above